config/main.py

```diff
@@ -1711,20 +1761,55 @@ def startup():
 @startup.command()
 @click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
 def all(verbose):
-    """Start up all BGP sessions"""
+    """Start up all BGP sessions
+       In the case of Multi-Asic platform, we startup only the EBGP sessions with external neighbors.
+    """
     log_info("'bgp startup all' executing...")
-    bgp_neighbor_ip_list = _get_all_neighbor_ipaddresses()
-    for ipaddress in bgp_neighbor_ip_list:
-        _change_bgp_session_status(ipaddress, 'up', verbose)
+    namespaces = [DEFAULT_NAMESPACE]
+    int_hosts = []
+
+    if is_multi_asic:
+        ns_list = get_all_namespaces()
+        namespaces = ns_list['front_ns']
+        int_hosts = get_all_internal_hosts()
+
+    """Connect to CONFIG_DB in linux host (in case of single ASIC) or CONFIG_DB in all the
+       namespaces (in case of multi ASIC) and do the sepcified "action" on the BGP neighbor(s)
+    """
+    for namespace in namespaces:
+        config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+        config_db.connect()
+        bgp_neighbor_ip_list = _get_all_neighbor_ipaddresses(config_db, int_hosts)
+        for ipaddress in bgp_neighbor_ip_list: 
+            _change_bgp_session_status_by_addr(config_db, ipaddress, 'up', verbose)
 
 # 'neighbor' subcommand
 @startup.command()
 @click.argument('ipaddr_or_hostname', metavar='<ipaddr_or_hostname>', required=True)
 @click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
 def neighbor(ipaddr_or_hostname, verbose):
-    """Start up BGP session by neighbor IP address or hostname"""
     log_info("'bgp startup neighbor {}' executing...".format(ipaddr_or_hostname))
-    _change_bgp_session_status(ipaddr_or_hostname, 'up', verbose)
+    """Start up BGP session by neighbor IP address or hostname.
+       User can specify either internal or external BGP neighbor to startup
+    """
+    namespaces = [DEFAULT_NAMESPACE]
+    found_neighbor = False
+
+    if is_multi_asic:
+        ns_list = get_all_namespaces()
+        namespaces = ns_list['front_ns'] + ns_list['back_ns']
+
+    """Connect to CONFIG_DB in linux host (in case of single ASIC) or CONFIG_DB in all the
+       namespaces (in case of multi ASIC) and do the sepcified "action" on the BGP neighbor(s)
+    """
```

Comment:
Prefer to reserve`"""` for docstrings and use `#` for comments

Reply:
Done.